Task/Taskfile.yaml

```diff
@@ -0,0 +1,10 @@
+# https://taskfile.dev
```

Comment:
Can we rename to Taskfile.yml?  I will hate my life forever if we don't nip this in the bud

Reply:
Sure thing, when you init a new Taskfile it names it "Taskfile.yaml" but I checked the style guide and it has "Taskfile.yml".